include/knowhere/prometheus_client.h

```diff
@@ -55,22 +55,35 @@ class PrometheusClient {
 extern const prometheus::Histogram::BucketBoundaries defaultBuckets;
 extern const std::unique_ptr<PrometheusClient> prometheusClient;
 
-#define DEFINE_PROMETHEUS_GAUGE(name, desc)                                                                  \
-    prometheus::Family<prometheus::Gauge>& name##_family =                                                   \
-        prometheus::BuildGauge().Name(#name).Help(desc).Register(knowhere::prometheusClient->GetRegistry()); \
+#define KNOWHERE_LABEL "knowhere"
+
+#define DEFINE_PROMETHEUS_GAUGE(name, desc, label)                                                                   \
```

Comment:
use another name than label(maybel "module"?), label refers to a prometheus label